AsyncDisplayKit/TextKit/ASTextKitRenderer+TextChecking.mm

```diff
@@ -75,6 +74,7 @@ - (NSTextCheckingResult *)textCheckingResultAtPoint:(CGPoint)point
     truncationTokenRange = { 0, truncationAttributedString.length };
   }
 
+  NSRange visibleRange = self.truncater.firstVisibleRange;
   truncationTokenRange.location += NSMaxRange(visibleRange);
```

Comment:
@maicki yes definitely, this is why I complement the design choice. I agree that the alternative is less elegant and thoughtful, but the reason if comes to mind is that it may be slightly more practical, in the sense that these values are used internally only and are not a public API and that the distinction between 0, 0 and NSNotFound is meaningless in this case… sept many easy to imagine usages of the value would have to explicitly check for the not found to avoid an invalid calculation that would otherwise be a safe non-operation with zero.

I assume NSMaxRange returns 0 for this case?